presto-hive/src/test/java/com/facebook/presto/hive/parquet/ParquetTester.java

```diff
@@ -182,7 +183,7 @@ private static void assertFileContents(JobConf jobConf,
         Iterator<?> iterator = expectedValues.iterator();
         for (int batchSize = parquetReader.nextBatch(); batchSize >= 0; batchSize = parquetReader.nextBatch()) {
             ColumnDescriptor columnDescriptor = fileSchema.getColumns().get(0);
-            Block block = parquetReader.readPrimitive(columnDescriptor, type);
+            Block block = parquetReader.readPrimitive(columnDescriptor, type, new ArrayList<>(), new ArrayList<>());
```

Comment:
Can you port over the struct, list and map tests from the ORC or RCFile tester code?